game.py

```diff
@@ -64,19 +66,29 @@ def put_down_card_on_table(self, player, card):
 
     def try_playing_matching_number_or_color_card(self, player):
         for card in player.cards:
-            if card.color == self.card_on_table.color or card.num == self.card_on_table.num:
+            if (card.color == self.card_on_table.color or card.num == self.card_on_table.num) and \
+                    not card.is_stop_card and not card.is_change_color:
+                self.put_down_card_on_table(player, card)
+                return True
+        return False
+
+    def try_playing_stop_card(self, player):
+        for card in player.cards:
+            if card.is_stop_card and card.color == self.card_on_table.color:
                 self.put_down_card_on_table(player, card)
+                self.block_the_next_player()
                 return True
         return False
 
     def try_playing_change_color_card(self, player):
+        colors = [color.name for color in Color]
```

Comment:
give Color some AVAILABLE_COLORS attribute as a const and use it in all places like this